test/core/address_utils/sockaddr_utils_test.cc

```diff
@@ -206,6 +209,33 @@ TEST(SockAddrUtilsTest, SockAddrToString) {
   EXPECT_TRUE(grpc_sockaddr_to_uri(&phony).empty());
 }
 
+#ifdef GRPC_HAVE_UNIX_SOCKET
+
+grpc_resolved_address MakeAddrUnix(std::string path) {
```

Comment:
Instead of writing your own function to do this, please use [`UnixSockaddrPopulate()`](https://github.com/grpc/grpc/blob/ab2193d8c178c41d2b097dc8b35a54bfdd9e928b/src/core/lib/address_utils/parse_address.h#L72) and [`UnixAbstractSockaddrPopulate()`](https://github.com/grpc/grpc/blob/ab2193d8c178c41d2b097dc8b35a54bfdd9e928b/src/core/lib/address_utils/parse_address.h#L77).

Reply:
Done